src/test/java/spoon/test/prettyprinter/PrinterTest.java

```diff
@@ -411,6 +412,9 @@ private void checkRepeatingOfTokens(String tokenType) {
 			//contract: each printed character is handled by listener. PrinterHelper is not called directly
 			//and because PrinterTokenListener above does not use PrinterHelper, the result must be empty
 			assertEquals(0, withEmptyListenerResult.length());
+			
+			//contract: result built manually from tokens is same like the one made by DefaultTokenWriter
+			assertEquals(standardPrintedResult, allTokens.toString());
```

Comment:
this contract is not required for now IMHO
I'd remove it and proceed with merge

Reply:
ok, just comment it please